src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.dto.FruitTransactionDto;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.interfaces.FileReader;
+import core.basesyntax.service.interfaces.FileWriter;
+import core.basesyntax.service.interfaces.FruitReportCreate;
+import core.basesyntax.service.interfaces.TransactionParser;
+import core.basesyntax.service.serviceimpl.FileReaderImpl;
+import core.basesyntax.service.serviceimpl.FileWriterImpl;
+import core.basesyntax.service.serviceimpl.FruitReportCreateImpl;
+import core.basesyntax.service.serviceimpl.FruitTransactionParser;
+import core.basesyntax.service.strategy.OperationHandler;
+import core.basesyntax.service.strategy.strategyimpl.BalanceOperation;
+import core.basesyntax.service.strategy.strategyimpl.PurchaseOperation;
+import core.basesyntax.service.strategy.strategyimpl.ReturnOperation;
+import core.basesyntax.service.strategy.strategyimpl.SupplyOperation;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String OPEN_FROM_FILE = "src/main/resources/fruitts.csv";
+    private static final String SAVE_TO_FILE = "src/main/resources/report.csv";
```

Comment:
Constants should be nouns, not verbs
```suggestion
    private static final String FROM_FILE = "src/main/resources/fruitts.csv";
    private static final String TO_FILE = "src/main/resources/report.csv";
```